tests/functional/kvpy/tombspan.py

```diff
@@ -18,6 +18,78 @@ def check_keys(cursor: hse.KvsCursor, expected: List[bytes]):
     for x, y in zip(expected, actual):
         assert x == y
 
+def test_basic(kvdb: hse.Kvdb, kvs: hse.Kvs):
+    kvs.put(b"a", b"1")
+    kvs.put(b"b", b"2")
+    kvs.put(b"c", b"3")
+    kvs.put(b"d", b"4")
+    kvs.put(b"e", b"5")
+    kvs.put(b"f", b"6")
+    kvs.put(b"g", b"7")
+    kvs.put(b"h", b"8")
+
+    cursor = kvs.cursor()
+    check_keys(cursor, [b"a", b"b", b"c", b"d", b"e", b"f", b"g", b"h"])
+
+    kvs.delete(b"a")
+    kvs.delete(b"b")
+    kvs.delete(b"c")
+    kvs.delete(b"d")
+    kvs.delete(b"e")
+
+    cursor.update_view()
+    cursor.seek(b"a")
+    kv = cursor.read()
+    assert kv == (b"f", b"6")
+    cursor.seek(b"a")
+    kv = cursor.read()
+    assert kv == (b"f", b"6")
+
+    kvs.delete(b"f")
+    cursor.update_view()
+    cursor.seek(b"a")
+    kv = cursor.read()
+    assert kv == (b"g", b"7")
+    cursor.seek(b"a")
+    kv = cursor.read()
+    assert kv == (b"g", b"7")
+
+    kvs.put(b"c", b"33")
+    cursor.update_view()
+    cursor.seek(b"a")
+    kv = cursor.read()
+    assert kv == (b"c", b"33")
+
+    cursor.destroy()
+
+'''
+Perform the operations as done by mongo when it's running range delete after having moved chunks
+across shards
+'''
+def tombspan_test(kvdb: hse.Kvdb, kvs: hse.Kvs):
+    numkeys = 100 * 1000
+    half = int(numkeys / 2)
```

Comment:
This cast seems unnecessary.

Reply:
Hmm....Strangely enough it was interpreting this as a float and wouldn't work without the cast, but seems to work jsut fine now. I'll remove it.

Reply:
Oh interesting. Good to know.